package.json

```diff
@@ -68,7 +69,7 @@
     "start": "node server.js",
     "start-docs": "npm run production && npm run docs && jekyll serve -w",
     "lint": "eslint js test bench && documentation --lint",
-    "test": "npm run lint && istanbul test --include-pid --report none --print none prova -- test/js/*/*.js -q",
+    "test": "npm run lint && istanbul test --include-pid --report none --print none tape -- test/js/*/*.js | tap-dot",
```

Comment:
I've had consistently bad experiences with piping tap output to some other process. It invariably breaks crash logging, `console.log`, or something else.

Can we use tap and one of its built-in reporters instead?